SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -325,7 +325,9 @@ Quarks {
 			if(fetch, { this.prFetchDirectory });
 			this.prReadDirectoryFile(dirTxtPath);
 		}.try({ arg err;
-			("Failed to read quarks directory listing: % %".format(if(fetch, directoryUrl, dirTxtPath), err)).error;
+			var text = err.tryPerform(\errorString);
+			if(text.isNil) { text = err.asString };
+			("Failed to read quarks directory listing: % %".format(if(fetch, directoryUrl, dirTxtPath), text)).error;
```

Comment:
seems like this line might get too long, would it be a good idea to put an `\n` between the two `%`'s?

Reply:
Addressed.